scripts/lib/cli-utils.mjs

```
@@ -0,0 +1,91 @@
+/**
+ * Escape a string for use in double-quoted YAML.
+ * Handles backslashes, quotes, newlines, tabs, and control characters.
+ */
+export function escapeYamlString(str) {
+  return str
+    .replace(/\\/g, '\\\\')     // Backslashes first (before other escapes add more)
+    .replace(/"/g, '\\"')       // Double quotes
+    .replace(/\n/g, '\\n')      // Newlines
+    .replace(/\r/g, '\\r')      // Carriage returns
+    .replace(/\t/g, '\\t')      // Tabs
+    .replace(/[\x00-\x08\x0B\x0C\x0E-\x1F]/g, ''); // Strip other control chars
```

Biome (2.3.13)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)